cmd/massastation/main.go

```diff
@@ -80,7 +80,7 @@ func main() {
 		server.Stop()
 	})
 	stationGUI.Lifecycle().SetOnStarted(func() {
-		server.Start(networkManager, pluginManager)
+		server.Start(networkManager, pluginManager, configDir)
```

Comment:
why passing this configDir everywhere ? instead of calling `configuration.Path()` where you need it ?

Reply:
because I can't import int package into pkg packages